runners/gearpump/pom.xml

```diff
@@ -64,11 +64,18 @@
                 </goals>
                 <configuration>
                   <groups>org.apache.beam.sdk.testing.RunnableOnService</groups>
+                  <excludedGroups>
```

Comment:
I think if you change `<activeByDefault>` to `true` then you should get ROS coverage in presubmit.